test/IntegrationTests/Microsoft.Testing.Platform.Acceptance.IntegrationTests/Runner.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Testing.Platform.Acceptance.IntegrationTests;
+
+[TestGroup]
+public class RunnerTests : BaseAcceptanceTests
+{
+    private readonly AcceptanceFixture _acceptanceFixture;
+    private const string AssetName = "MSTestProject";
+
+    public RunnerTests(ITestExecutionContext testExecutionContext, AcceptanceFixture acceptanceFixture)
+        : base(testExecutionContext, acceptanceFixture)
+    {
+        _acceptanceFixture = acceptanceFixture;
+    }
+
+    [ArgumentsProvider(nameof(GetBuildMatrixTfmBuildConfiguration), TestArgumentsEntryProviderMethodName = nameof(FormatGetBuildMatrixTfmBuildConfigurationEntry))]
+    public async Task EnableMSTestRunner_True_Will_Run_Standalone(string tfm, BuildConfiguration buildConfiguration)
+    {
+        using TestAsset generator = await TestAsset.GenerateAssetAsync(
+            AssetName,
+            CurrentTemplateSourceCode
+            .PatchCodeWithRegularExpression("tfm", tfm)
+            .PatchCodeWithRegularExpression("mstestversion", MSTestCurrentVersion)
+            .PatchCodeWithRegularExpression("enablemstestrunner", "<EnableMSTestRunner>true</EnableMSTestRunner>")
+            .PatchCodeWithRegularExpression("outputtype", "<OutputType>Exe</OutputType>"),
+            addPublicFeeds: true);
+        string binlogFile = Path.Combine(generator.TargetAssetPath, "msbuild.binlog");
+        var compilationResult = await DotnetCli.RunAsync($"restore -nodeReuse:false {generator.TargetAssetPath} -r {RID}", _acceptanceFixture.NuGetGlobalPackagesFolder);
+        compilationResult = await DotnetCli.RunAsync(
+            $"build -nodeReuse:false {generator.TargetAssetPath} -c {buildConfiguration} -bl:{binlogFile} -r {RID}",
+            _acceptanceFixture.NuGetGlobalPackagesFolder, failIfReturnValueIsNotZero: false);
+        var testHost = TestInfrastructure.TestHost.LocateFrom(generator.TargetAssetPath, AssetName, tfm, buildConfiguration: buildConfiguration);
+        var testHostResult = await testHost.ExecuteAsync(environmentVariables: new Dictionary<string, string>() { { "TESTINGPLATFORM_TELEMETRY_OPTOUT", "1" } });
+        Assert.IsTrue(testHostResult.StandardOutput.Contains("Passed! - Failed: 0, Passed: 1, Skipped: 0, Total: 1"), $"Standard output:\n'{testHostResult.StandardOutput}'");
```

Comment:
Please use the assert helper:

```suggestion
        testHostResult.AssertOutputContains("Passed! - Failed: 0, Passed: 1, Skipped: 0, Total: 1");
```